lib/tiki_sdk.dart

```diff
@@ -32,23 +31,31 @@ class TikiSdk {
   final LicenseService _licenseService;
   final NodeService _nodeService;
 
-  /// Recommended to use [init] instead.
+  /// Recommended to use [withAddress] and [init] instead.
   TikiSdk(TitleService titleService, LicenseService licenseService,
       NodeService nodeService)
       : _titleService = titleService,
         _licenseService = licenseService,
         _nodeService = nodeService;
 
-  static init(String publishingId, String orign, KeyStorage keyStorage,
-      String databaseDir,
+  static Future<String> withAddress(KeyStorage keyStorage,
```

Comment:
@jcockhren, just FYI. 

I split the initialization into 2 parts, so the platform sdks can pass in any CommonDatabase.